engine/src/main/java/org/terasology/rendering/opengl/LwjglRenderingProcess.java

```diff
@@ -163,7 +155,12 @@
     /* VARIOUS */
     private boolean takeScreenshot;
     private int displayListQuad = -1;
+
+    // Note: this assumes that the settings in the configs might change at runtime,
+    // but the config objects will not. At some point this might change, i.e. implementing presets.
     private Config config = CoreRegistry.get(Config.class);
+    private RenderingConfig renderingConfig = config.getRendering();
```

Comment:
I think that a simple getter does not affect runtime performance, in particular if it's called only once per frame.
So I guess, you could leave it in the method call, so you don't need to rely on getRendering() to be constant.

Imagine a "Reset Video Settings" button, which would re-load the Config from file or create a new one.